README.md

```diff
@@ -11,6 +11,16 @@ NeoVim is supported.
 - all open viewers automatically scroll to keep in sync with your cursor in Vim
 - open the current buffer's contents in vivify with `:Vivify`
 
+## Configuration
+
+- `g:vivify_instant_refresh = 1`
+    - refresh page contents on `TextChanged` and `TextChangedI` (default)
+- `g:vivify_instant_refresh = 0`
+    - refresh page contents on `CursorHold` and `CursorHoldI`
+- `g:vivify_filetypes`
+    - additional filetypes to recognize as markdown
+    - for example `let g:vivify_filetypes = ['vimwiki']`
```

Comment:
This is not quite accurate: It specifies which file types will make Vivify live-sync the viewer. Doesn't necessarily have to be Markdown. Let's also specify that this is `'markdown'` by default.

Reply:
Actually, I think putting other filetypes here would not be smart because it would update them without the code-highlighting wrapper ^^ So forget about the first part, let's just only include the default setting here.

Reply:
Okay sorry, I think how it is now is good. No need for the default setting since it's **additional** filetypes and doesn't affect the default.